package.json

```diff
@@ -32,15 +32,15 @@
     "rewire": "^2.5.2"
   },
   "scripts": {
-    "test": "node_modules/jest/bin/jest.js --config jest.config.json",
-    "test:watch": "node_modules/jest/bin/jest.js --watch --config jest.config.json",
-    "coverage": "jest --coverage && google-chrome ./coverage/lcov-report/index.html --config jest.config.json",
+    "test": "node_modules/jest/bin/jest.js",
+    "test:watch": "node_modules/jest/bin/jest.js --watch",
+    "coverage": "jest --coverage && google-chrome ./coverage/lcov-report/index.html",
     "start": "nodemon src/server.js --exec babel-node",
     "eslint": "node_modules/eslint/bin/eslint.js -c .eslintrc.json src/**",
     "db": "mongod --dbpath c:/dev/mongo/data",
     "open": "concurrently \"mongod --dbpath c:/dev/mongo/data\" \"nodemon server.js\"",
     "clean": "rm -rf dist && mkdir dist",
-    "build": "./node_modules/babel-cli/bin/babel.js src -d dist",
+    "build": "./node_modules/babel-cli/bin/babel.js src -d dist --source-maps",
```

Comment:
Adding source maps here so that vscode can follow your breakpoints even after they've been transpiled.